go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/google/uuid v1.4.0
 	github.com/kuadrant/authorino v0.15.0
 	github.com/kuadrant/authorino-operator v0.9.0
-	github.com/kuadrant/dns-operator v0.1.0
+	github.com/kuadrant/dns-operator v0.0.0-20240321074453-b11e4521d05d
```

Comment:
@philbrookes fyi, since we aren't changing the module name, decided it was probably easier to just do it this way. Can update with:

```
go get github.com/kuadrant/dns-operator@main
```